### PR TITLE
CLI: add prerelease warning

### PR DESCRIPTION
**Pull request by natemoo-re**
## Changes

- Adds a warning on dev startup when Astro is in a pre-release mode.

Screen Shot 2022-03-10 at 4 58 12 PM

## Testing

Tested manually, we need snapshot tests to properly cover this

## Docs

N/A but added Netlify redirect for https://astro.build/issues

### CODE REVIEW

**Comment by FredKSchott** [comment]
Feedback:

- This is gorgeous!
- re `prerelease` I like this, but wish the second message ("Undocumented changes may happen at any time") was less scary or maybe didn't exist at all. Could it instead say something productive like `"Leave feedback and report bugs at astro.build/error"` ?
- re `update available` This feels like one of those classic "oh this sounds cool but lets think it through a bit more":
  - This means that startup will hang out an API call every time? I'm at a coffeeshop now and I can just imagine how frustrating that would be.
  - If I'm not an older version for a reason, I could see this also being annoying. the most popular npm package for handling this for you will give you a max of 1 warning a day (maybe week?) for that reason.

File: packages/astro/src/core/messages.ts
Comment (FredKSchott) [inline packages/astro/src/core/messages.ts:65]: super duper nit, only if you like it: 

```
Feedback? https://astro.build/issues
```

Also, this means we need to actually create this URL before deploying :) Feel free to use https://github.com/withastro/astro/issues/new/choose so that you're not blocked on merging this today, and then we can add a vanity URL later.

**Review by FredKSchott — APPROVED** [review]
Approved once whatever URL you use actually exists!

File: packages/astro/src/core/messages.ts
Comment (natemoo-re) [reply packages/astro/src/core/messages.ts]: Yep, I like that. 👍🏻 
URL already works! https://astro.build/issues

**Pull request merged**
